src/components/protocol_handler/include/protocol_handler/protocol_packet.h

```diff
@@ -103,12 +103,14 @@ class ProtocolPacket {
     void set_max_rpc_payload_size(const size_t max_payload_size);
     void set_max_audio_payload_size(const size_t max_payload_size);
     void set_max_video_payload_size(const size_t max_payload_size);
+    void set_max_protocol_version_supported(const uint8_t max_payload_size);
 
     size_t max_payload_size() const;
     size_t max_control_payload_size() const;
     size_t max_rpc_payload_size() const;
     size_t max_audio_payload_size() const;
     size_t max_video_payload_size() const;
+    size_t max_protocol_version_supported() const;
```

Comment:
@SKobziev i think that the return type should correspond the type of variable you are returning

Reply:
It's corrected.